### ISSUE TITLE
reshare v1.0: rf_15 & rf_17 grid spy files are in archive and should not be

### ISSUE THREAD

**Issue by dataknut** [CfSOtago/GREENGridData#19]
We split 15 & 17 into 15a/15b & 17a/17b due to re-use of monitor. Users should remove:

- rf_15_all_1min_data.csv.gz 
- rf_17_all_1min_data.csv.gz 

from the data archive if they have them.

We will remove it from future releases.

**Comment by raffertyparker** [comment]
>We split 15 & 17 into 15a/15b & 17a/17b due to re-use of monitor
There's no 15a (only 15 and 15b). Should 15 still be removed?

**Comment by dataknut** [comment]
yes, remove 15. 15a produced no data (or at least no data that was acceptable to the cleaning scripts :-)
